templates/_language_picker.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+{% get_current_language as LANGUAGE_CODE %}
+{% get_language_info for LANGUAGE_CODE as current_lang %}
+{% get_available_languages as LANGUAGES %}
+{% get_language_info_list for LANGUAGES as languages %}
+
+{# We display language picker only if there're languages to pick from #}
+{% if languages|length > 1 %}
```

Comment:
Isn't `{% if languages %}` enough here?

Reply:
I thought that someone may have 1 language defined inside of `LANGUAGES`, so that's why I'm checking, if there's something to pick from

Reply:
You're right, I misunderstood it at first read.